dnf-behave-tests/features/dnf-transaction-exception.feature

```diff
@@ -0,0 +1,23 @@
+@no_installroot
+Feature: Test exception when dnf tries to do a transaction resolved but not valid anymore
+
+Background: setup repository
+  Given I use repository "dnf-transaction-exception"
```

Comment:
> I wonder if we want to make this a repository dedicated to these utility plugins...
Good idea

> What about naming the repository e.g. utility-plugins as a place for these helper plugins in case we have more of these in the future?
`utility-plugins` is surely better

> I found the watermelons grin in fixtures/specs/plugins-callbacks/*
`fixtures/specs/plugins-callbacks/*` is actually the folder from which I took the example for this code :)

Reply:
The name for the repo is now, `dnf-transaction-exception -> utility-plugins`: do you think the name of the feature-file should be changed?
Maybe now I am overthinking the problem but...
Should the feature-file be moved in a subdir as well such as `features/utility-plugins/dnf-transaction-exception.feature`?